relay-kafka/src/config.rs

```diff
@@ -156,8 +167,7 @@ impl Default for TopicAssignments {
             outcomes: "outcomes".to_owned().into(),
             outcomes_billing: None,
             sessions: "ingest-sessions".to_owned().into(),
-            metrics: "ingest-metrics".to_owned().into(),
```

Comment:
I removed this because I think it's confusing / error prone. There shouldn't be a metrics default. Only 2 completely separate topics for either release health metrics and generic metrics.